src/styles/components/_card.scss

```diff
@@ -0,0 +1,164 @@
+@import "../abstracts/_variables.scss";
+@import "../base/base";
+
+@media only screen and (max-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(1, 1fr);
+    max-width: $card-width;
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 488px) {
+  .cards {
+    grid-template-columns: repeat(2, 1fr);
+    max-width: calc(2 * #{$card-width} + 1 * #{$horizontal-gap});
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .cards {
+    grid-template-columns: repeat(3, 1fr);
+    max-width: calc(3 * #{$card-width} + 2 * #{$horizontal-gap});
+    margin: 0 auto;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .cards {
+    grid-template-columns: repeat(4, 1fr);
+    max-width: calc(4 * #{$card-width} + 3 * #{$horizontal-gap});
+    margin: 0 auto;
+  }
+}
+
+.cards {
+  display: grid;
+  padding: $cards-padding;
+  gap: $vertical-gap $horizontal-gap;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: $card-padding;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  box-sizing: border-box;
+  z-index: 1;
+  transition: 300ms;
+
+  &__description {
+    width: 166px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    z-index: 2;
+    transform: scale(1.2);
+  }
+
+  &:hover &__description {
+    color: $description-hover-color;
+  }
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    background-image: url("../../src/images/imac.jpeg");
```

Comment:
Add this image as `img` tag, and not the background image, this image is a vital part of a `card`, so it should be placed in HTML, it would be better for accessibility, also this `background-image` may cause some of your tests to fail.